diff --git a/dist/esm/staticFunctionMiddleware.js b/dist/esm/staticFunctionMiddleware.js
index 3f9b7db31b55634034ac348ef8812b62aa0ff922..12c7a6f7f53765b54ded4b90675b3fc900a170ac 100644
--- a/dist/esm/staticFunctionMiddleware.js
+++ b/dist/esm/staticFunctionMiddleware.js
@@ -53,7 +53,7 @@ var fetchItem = async ({ data, functionId }) => {
 		functionId,
 		hash: jsonToFilenameSafeString(data)
 	});
-	return fetch(url, { method: "GET" }).then((r) => r.json()).then((d) => fromJSON(d, { plugins: getSerovalPlugins(defaultSerovalDeserializerPlugins) }));
+	return fetch(import.meta.env.BASE_URL.replace(/\/$/, "") + url, { method: "GET" }).then((r) => r.json()).then((d) => fromJSON(d, { plugins: getSerovalPlugins(defaultSerovalDeserializerPlugins) }));
 };
 var staticFunctionMiddleware = createMiddleware({ type: "function" }).client(async (ctx) => {
 	if (process.env.NODE_ENV === "production" && typeof document !== "undefined") {
